### PR TITLE
EDD-6: Adds redirect for Earthdata Download auth callback

### PR DESCRIPTION
**Pull request by macrouch**
# Overview

### What is the feature?

Adding a redirect for Earthdata Download when a EDD user needs to login to get a new token.

# Testing

### Reproduction steps

Submit a project with a collection with the download access method. Click on Download Files to follow the EDD workflow, EDD should open a browser and redirect to the EDSC login page. After logging in EDSC will redirect you back to EDD to continue downloading

# Checklist

- [x] I have added automated tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

File: serverless/src/edlCallback/__tests__/handler.test.js [hunk 1]
```diff
@@ -58,8 +66,8 @@ beforeEach(() => {
   dbTracker = mockKnex.getTracker()
   dbTracker.install()
 
-  SQSClient.mockClear()
-  SendMessageCommand.mockClear()
+  // SQSClient.mockClear()
```
Comment (eudoroolivares2016) [inline serverless/src/edlCallback/__tests__/handler.test.js:69]: remove

File: static/src/js/reducers/__tests__/earthdataDownloadRedirect.test.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
+import earthdataDownloadRedirectReducer from '../earthdataDownloadRedirect'
+import { ADD_EARTHDATA_DOWNLOAD_REDIRECT } from '../../constants/actionTypes'
+
+describe('INITIAL_STATE', () => {
+  test('is correct', () => {
```
Comment (eudoroolivares2016) [inline static/src/js/reducers/__tests__/earthdataDownloadRedirect.test.js:5]:
```suggestion
  test('dummy action type returns initial state', () => {
```

File: static/src/js/routes/EarthdataDownloadRedirect/EarthdataDownloadRedirect.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+export const EarthdataDownloadRedirect = () => {
+  const { edscHost } = getEnvironmentConfig()
+
+  return (
+    <>
+      <Helmet>
+        <title>Earthdata Download Redirect</title>
+        <meta name="title" content="Earthdata Download Redirect" />
+        <meta name="robots" content="noindex, nofollow" />
```
Comment (eudoroolivares2016) [inline static/src/js/routes/EarthdataDownloadRedirect/EarthdataDownloadRedirect.js:17]: Do we need this robots line? I removed it from EarthdataDownload route or is this used for the future google analytics tickets we have?

**Review by eudoroolivares2016 — APPROVED** [review]

File: static/src/js/routes/EarthdataDownloadRedirect/EarthdataDownloadRedirect.js (hunk 3 above)
Comment (macrouch) [reply static/src/js/routes/EarthdataDownloadRedirect/EarthdataDownloadRedirect.js:17]: @trevorlang ?

**Pull request merged**